.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
If you want to edit your VSCode settings maybe toss that in the .gitignore by including the .vscode folder in that file